src/routes/establishment/create.js

```diff
@@ -0,0 +1,66 @@
+import uuidv4 from 'uuid/v4';
+import Establishment from '../../models';
+
+/**
+ * @swagger
```

Comment:
very nice swagger specs btw. Do you use a generator for these? I do these by hand and it can get tedious lol